assets/queries/openAPI/response_object_incorrect_ref/query.rego

```diff
@@ -6,7 +6,7 @@ CxPolicy[result] {
 	doc := input.document[i]
 	openapi_lib.check_openapi(doc) != "undefined"
 	response := doc.paths[n][oper].responses[code]
-
+	openapi_lib.content_allowed(oper, code)
```

Comment:
There is a reason to check if content is allowed here? Since it is checking reference, I don't see relation with content here...

Reply:
No content should be allowed for HEAD responses